manifests/init.pp

```diff
@@ -35,21 +33,26 @@
         default => 'absent'
       }
 
-      kernel_parameter {
-        'fips':
-          value  => $fips_kernel_value,
-          notify => Reboot_notify['fips'];
-          # bootmode => 'normal', # This doesn't work due to a bug in the Grub Augeas Provider
-        'boot':
-          value  => "UUID=${::boot_dir_uuid}",
+      kernel_parameter { 'fips':
+        value  => $fips_kernel_value,
+        notify => Reboot_notify['fips']
+        # bootmode => 'normal', # This doesn't work due to a bug in the Grub Augeas Provider
+      }
+
+      # This should only be present if /boot is on a separate partition
+      if $facts['boot_dir_uuid'] and ($facts['boot_dir_uuid'] != $facts['root_dir_uuid']) {
```

Comment:
@jeefberkey I'll go ahead and switch it. It's unfortunate because the data structure is straight up backwards from what I need, but I suppose that I was being a bit lazy here.

I'll go kill the PR for simplib that added the root fact as well.